src/world/light/mod.rs

```diff
@@ -6,7 +6,7 @@ use crate::graphics::*;
 #[cfg(feature = "debug")]
 use crate::world::MarkerIdentifier;
 
-#[derive(PrototypeElement, PrototypeWindow, new)]
+#[derive(ByteConvertable, PrototypeElement, PrototypeWindow, new)]
```

Comment:
I assume this is a left over from you trying to derive `ByteConvertable` for `LightSource`?

Reply:
LightSource is now ByteConvertable, so I removed the "new" trait